README.rst

```diff
@@ -167,6 +167,14 @@ If you are looking to start working with the Zipline codebase, navigate to the G
 
 Feel free to ask questions on the `mailing list <https://groups.google.com/forum/#!forum/zipline>`_ or on `Gitter <https://gitter.im/quantopian/zipline>`_.
 
+Please note that even if Zipline is `free software <https://en.wikipedia.org/wiki/Free_software>`_, it is not a community-lead project and we are a busy team. These are all possible scenarios that you should consider when contributing:
+
+- Your pull-request may be ignored for months, or even years, without any kind of feedback
+- Your pull-request may be closed after we implement a different kind, or even the same solution on our own, discarding your changes
+- We may not be able to respond to direct @mentions on GitHub
+
+We hope you understand and we hope this information reduces possible frustration and wasted time for everybody.
+
```

Comment:
@richafrank Thanks for your reply. :blush: 

A couple of comments:

- "Timely manner" seems a bit too soft to represent months or years of wait
- Making it clear that the project is not community-lead seems important to me

My new proposal:

> Please note that Zipline is not a community-lead project. Zipline is maintained by the Quantopian engineering team, and we are quite small and often busy. Because of this, we want to warn you that we may not attend to your pull request, issue, or direct mention in months, or even years. We hope you understand, and we hope that this note might help reduce any frustration or wasted time

Reply:
Points taken! I just corrected some typos:
```suggestion
.. note::

    Please note that Zipline is not a community-led project. Zipline is maintained by the Quantopian engineering team, and we are quite small and often busy.
    
    Because of this, we want to warn you that we may not attend to your pull request, issue, or direct mention in months, or even years. 
    We hope you understand, and we hope that this note might help reduce any frustration or wasted time.
```

Reply:
@richafrank Thanks for your consideration. :heart:

I just force-pushed the changes (https://github.com/quantopian/zipline/pull/2558/commits/3c7393c3cb8d21b30784f1978c6ada00c8886753) and rebased against `upstream/master`.